### PR TITLE
Skip selected participants in Attendance

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- User can select participants to not create Service Deliveries for in the Attendance component.

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- ~~Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))~~
- ~~Base axe-core JEST test included for any new LWC (No critical violations)~~
- ~~CRUD/FLS is enforced in Apex~~
- ~~Permission sets & field sets are updated to account for CRUD/FLS/TABS for new object metadata~~
- ~~All modified classes are unit tested (Apex) at 100% coverage~~
- [x] UX approval or UX not necessary
- [x] PR contains draft release notes
- [x] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [x] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by bethbrains** [comment]
[W-8535067](https://gus.my.salesforce.com/a07B0000008klaDIAQ)

**Review by lbeller — APPROVED** [review]

File: force-app/main/default/lwc/attendanceRow/attendanceRow.js [hunk 1]
```diff
@@ -102,4 +111,21 @@ export default class AttendanceRow extends LightningElement {
...
+    get showSkip() {
+        return !this.recordId && !this.readOnly;
+    }
+
+    handleToggleButton() {
+        this.rowDisabled = !this.rowDisabled;
+        if (this.rowDisabled) {
+            let inputFields = [
+                ...this.template.querySelectorAll("lightning-input"),
+                ...this.template.querySelectorAll("lightning-input-field"),
```
Comment (bmuniswamy) [inline force-app/main/default/lwc/attendanceRow/attendanceRow.js:124]: I did not know we could do something like this.

File: force-app/main/default/lwc/attendanceRow/attendanceRow.js [hunk 2]
```diff
@@ -102,4 +111,21 @@ export default class AttendanceRow extends LightningElement {
...
+            ];
+            inputFields.forEach(field => {
+                field.value = null;
```
Comment (bmuniswamy) [inline force-app/main/default/lwc/attendanceRow/attendanceRow.js:127]: One question here. Do we need to set the value to null here when we disable a row.

Comment (bethbrains) [reply force-app/main/default/lwc/attendanceRow/attendanceRow.js:127]: Yes, this was a late-breaking AC from Meg yesterday. it makes the UX much clearer what's happening. I think I added it to the ACs.

**Review by lbeller — APPROVED** [review]

**Review by bmuniswamy — APPROVED** [review]
Great work Beth !!!

**Review by lbeller — APPROVED** [review]

**Pull request merged**